app/assets/javascripts/validation.coffee

```diff
@@ -248,10 +248,17 @@ $(document).ready ->
       $(errorElement).insertAfter(afterElement)
 
   displaySummary = (errors) ->
-    summary = $('<div/>',
-      class: 'eui-banner--danger summary-errors'
-      html: '<h4><i class="fa fa-exclamation-triangle"></i> This draft has the following errors:</h4>'
-    )
+    # This modal is loaded on every new/edit page for templates, but not drafts
+    if $('#invalid-template-modal').length > 0
+      summary = $('<div/>',
+        class: 'eui-banner--danger summary-errors'
+        html: '<h4><i class="fa fa-exclamation-triangle"></i> This template has the following errors:</h4>'
+      )
+    else
+      summary = $('<div/>',
+        class: 'eui-banner--danger summary-errors'
+        html: '<h4><i class="fa fa-exclamation-triangle"></i> This draft has the following errors:</h4>'
```

Comment:
I'm wondering if we can make this a little more succinct by setting a variable (just an example but maybe `resource_type`) to 'template' or 'draft', then interpolating it into the html section (i.e. `"... This #{resource_type} ..."`)

Reply:
Done.  Didn't know coffeescript had included an alternate way of doing interpolation.  JS's backticks were not cooperating when I tried it originally.